packages/artifact/src/internal/crc64.ts

```diff
@@ -0,0 +1,300 @@
+/**
+ * CRC64: cyclic redundancy check, 64-bits
+ *
+ * In order to validate that artifacts are not being corrupted over the wire, this redundancy check allows us to
+ * validate that there was no corruption during transmission. The implementation here is based on Go's hash/crc64 pkg,
+ * but without the slicing-by-8 optimization: https://cs.opensource.google/go/go/+/master:src/hash/crc64/crc64.go
+ *
+ * This implementation uses a pregenerated table based on 0x9A6C9329AC4BC9B5 as the polynomial, the same polynomial that
+ * is used for Azure Storage: https://github.com/Azure/azure-storage-net
+ */
+
+// when transpile target is >= ES2020 (after dropping node 12) these can be changed to bigint literals - ts(2737)
+const AZURE_TABLE = [
+  BigInt('0x0000000000000000'),
```

Comment:
I found the original file, pretty cool: https://github.com/Azure/azure-storage-net/blob/cbe605f9faa01bfc3003d75fc5a16b2eaccfe102/Lib/Common/Core/Util/Crc64.cs#L29

How about linking to that directly in the comment?

Also i would change `AZURE_TABLE` to a different name, though I can't think of anything off the top of my head

Reply:
changed to `PREGEN_POLY_TABLE` in https://github.com/actions/toolkit/pull/1063/commits/b68735e06098244d8f6a11cfac4f0000d6807e4c